source/administration/notices.rst

```diff
@@ -43,7 +43,10 @@ Notices will be used to raise awareness of new features as part of our monthly r
 Admin Advisor notices
 ---------------------
 
-Admin Advisor notices are included in Mattermost from v5.26. They're designed to provide guidance and assist in choosing features to support your deployment as it scales.
+.. note::
```

Comment:
This may work better swapped around (i.e., the note beneath the "Admin Advisor notices..." sentence? Other than that, LGTM. :)